### ISSUE TITLE
Have a more detailled error when php-intl is missing

### ISSUE THREAD

**Issue by SebastienGautier** [zendframework/zend-i18n#16]
When the php-intl extension is not found, an error is thrown : 

```
Fatal error: Uncaught exception 'Zend\Mvc\Exception\BadMethodCallException' with message 'Unable to call method "addTranslationFile"; does not exist in translator' in [...]\vendor\zendframework\zendframework\library\Zend\Mvc\I18n\Translator.php:47
Stack trace:
#0 [...]\module\Application\Module.php(28): Zend\Mvc\I18n\Translator->__call('addTranslationF...', Array)
#1 [...]\module\Application\Module.php(28): Zend\Mvc\I18n\Translator->addTranslationFile('phpArray', './vendor/zendfr...', 'default', 'fr_FR')
#2 [internal function]: Application\Module->onBootstrap(Object(Zend\Mvc\MvcEvent))
#3 [...]\vendor\zendframework\zendframework\library\Zend\EventManager\EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#4 [...]\vendor\zendframework\zendframework\library\Zend\EventManag in [...]\vendor\zendframework\zendframework\library\Zend\Mvc\I18n\Translator.php on line 47
```

You should add php-intl in the composer requirements or at least set a more explicit error about php-intl missing.

Thanks !

**Comment by adamculp** [comment]
Perhaps the composer.json should contain the requirement for php-intl? This would be more clear, and ensure the php extension is installed.

**Comment by weierophinney** [comment]
@adamculp and @groseb Agreed - can one of you submit a PR for this, please?

**Comment by adamculp** [comment]
I'm on it.

**Comment by adamculp** [comment]
Added PR #126

**Comment by michalbundyra** [comment]
Closed by #126
